packages/medusa/src/commands/migrate.ts

```diff
@@ -20,21 +23,39 @@ const main = async function ({ directory }) {
   const plugins = getResolvedPlugins(directory, configModule, true) || []
   const pluginLinks = await resolvePluginsLinks(plugins, container)
 
-  if (args[0] === "run") {
+  if (action === "run") {
+    Logger.info("Running migrations...\n")
```

Comment:
Q: Do we need to add a new line since every call is on a new line?

Reply:
I want an empty line after the `Running migrations` log to visually break the migration logs from the process start logs. It will be clear once I will post screenshots